Chapter1.Rmd

```diff
@@ -4,36 +4,71 @@ Sections covered: all
 
 ## 1.1 Populations, Samples, and Processes {-}
 
+To generate a random sample of data, we use the function `sample()`. 
+
+**Ex:** To generate a sample of 6 numbers from the integers between 1 and 20:
+
+```{r}
+sample(x = 1:20, size  = 6)
+```
+
+Note: this syntax assumes that there is no replacement (it won't pick the same number twice).
+If you want to sample *with* replacement use the following:
+
+```{r}
+sample(x = 1:20, size = 8, replace = TRUE)
+```
+
 ## 1.2 Pictorial and Tabular Methods in Descriptive Statistics {-}
 
 Skip: Example 1.7, p. 15 (double-digit leaves)
 
 Skip: "Dotplots," pp. 15-16
 
+
 * Stem-and-leaf display:
+**Ex:** The following are the prices (in $) in a jewelry store:
```

Comment:
Please change to: listing prices of one-bedroom apartments in Morningside Heights in 2016 (in thousands).